### PR TITLE
migrate more samples

### PR DESCRIPTION
**Pull request by sebastianbenz**
* Rewrote amp-bind sample to be shorter and more sample focused.
* Moved a few samples to the respective component samples
* Fixed a indentation bug resulting in doc not being formatted

### CODE REVIEW

File: src/amphtml-email/10_Introduction/_Hello_World.html [hunk 1]
```diff
@@ -65,17 +65,15 @@
...
-      </p>
     -->
     <div class="images">
       <amp-img src="https://ampbyexample.com/img/amp.jpg" width="800" height="600" layout="responsive"></amp-img>
       <!--
         ### AMP Components
 
         The following AMP components are supported in AMPHTML email messages.
-        <h4>Dynamic Content </h4>
+        **Dynamic Content </h4>
```
Comment (kul3r4) [inline src/amphtml-email/10_Introduction/_Hello_World.html]: I missed this: remove the closing h4

File: templates/new-example.html [hunk 2]
```diff
@@ -40,9 +41,10 @@
 </head>
 <body>
 
-  <!-- ## Basic Usage -->
-  <!--
-  TODO: Describe your example here
+  <!-- 
+    ## Basic Usage
```
Comment (kul3r4) [inline templates/new-example.html]: Usage -> usage

**Comment by sebastianbenz** [comment]
@kul3r4 PTAL

I've started improving the parser (to avoid the case where docs are printed as code). This might still need more work. Will address in another PR.

File: src/amphtml-email/10_Introduction/_Hello_World.html (hunk 1 above)
Comment (sebastianbenz) [reply src/amphtml-email/10_Introduction/_Hello_World.html]: this is broken anyway and needs a complete refactor...plan to do this in another PR

**Pull request merged**